src/settings.rs

```diff
@@ -255,23 +267,23 @@ mod tests {
 
             let settings = Settings {
                 run_as_user: RuleStrategy {
-                    rule: String::from("RunAsAny"),
+                    rule: Rule::RunAsAny,
                     ranges: vec![IDRange {
                         min: 1000,
                         max: 1010,
                     }],
                     ..Default::default()
                 },
                 run_as_group: RuleStrategy {
-                    rule: String::from("RunAsAny"),
+                    rule: Rule::RunAsAny,
                     ranges: vec![IDRange {
                         min: 1000,
                         max: 1010,
                     }],
                     ..Default::default()
                 },
                 supplemental_groups: RuleStrategy {
-                    rule: String::from(rule),
+                    rule: Rule::MustRunAs,
```

Comment:
I think this is incorrect, shouldn't it be `rule: rule`, as in line 237?

On line 226, we iterate through all allowed rules values, and we create to possible settings, one where we change the `run_as_group`, and one where we change the `supplemental_groups`.